From a21bc5dc7cc6c707ecd69810c8283acb296cb8a5 Mon Sep 17 00:00:00 2001
From: Sandor Yu <Sandor.yu@nxp.com>
Date: Wed, 20 Apr 2016 18:16:30 +0800
Subject: [PATCH 1026/5242] MLK-12688-02: arm dts: Add csis-clk-settle
 property

commit  fd257b19364ceb2e1582f3d6541502671484afa6 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add csis-clk-settle property to imx7D SDB mipi csi.

Signed-off-by: Sandor Yu <Sandor.yu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-sdb.dts |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index f27efac..f3da7bf 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -329,6 +329,7 @@
 			remote-endpoint = <&ov5640_mipi_ep>;
 			data-lanes = <2>;
 			csis-hs-settle = <13>;
+			csis-clk-settle = <2>;
 			csis-wclk;
 		};
 
-- 
1.7.9.5

